### DISCUSSION TITLE
Docker Daemon fails to start with the camera due to slow SD Card mount

### DISCUSSION THREAD

**Discussion by LukasGestalt** [AxisCommunications/discussions#243] [ACAP]
Hi,
I am developing an ACAP application that is deployed on the camera in a docker container. The application is supposed to run whenever the camera is switched on. It seems, however, that the slow mounting of the SD Card causes the docker service to fail to start, so it needs to be started manually after the SD card has been mounted properly. Do you have any advice on how I could solve or circument this?

The camera model is Q1656-LE. Here is the service log:

> 2023-05-12T14:26:57.536+02:00 axis-b8a44f49780b [ INFO    ] dockerdwrapper[1497]: Started logging.
> 2023-05-12T14:26:57.574+02:00 axis-b8a44f49780b [ ERR     ] dockerdwrapper[1497]: Cannot store data on the SD card, no storage exists at /var/spool/storage/SD_DISK
> 2023-05-12T14:26:57.574+02:00 axis-b8a44f49780b [ ERR     ] dockerdwrapper[1497]: Couldn't identify the file system of the SD card at /var/spool/storage/SD_DISK
> 2023-05-12T14:26:57.605+02:00 axis-b8a44f49780b [ ERR     ] dockerdwrapper[1497]: Starting dockerd failed
> 2023-05-12T14:26:57.617+02:00 axis-b8a44f49780b [ INFO    ] dockerdwrapper[1497]: Shutting down. dockerd shut down successfully.

 Thank you!

**Comment by Corallo** [comment]
HI @LukasGestalt 

I'd suggest trying with another "newer" SD card. We have noticed in the past similar problems when using a slow SD card.

**Reply by LukasGestalt** [reply]
Hi @Corallo, thanks for the advice.

We're using a new Samsung Evo Plus A2 512GB card, I don't think that's particularly slow. Could there be a way to ensure that the docker daemon waits for the mounting to finish?

**Reply by Corallo** [reply]
I see, we would have to try to reproduce this issue. 
Can you provide some "minimal reproducible instructions" on how you get to it?
Is the failure consistent? Are you using the latest firmware?

**Reply by marbali8** [reply]
Hey! It would be very helpful if you could also include the full device server report or syslog.
